deadpool.py

```diff
@@ -630,6 +631,7 @@ def raw_runner2(
     mem_clear_threshold_bytes: Optional[int] = None,
     kill_proc_tree=kill_proc_tree,
 ):
+    setproctitle("deadpool.worker")
```

Comment:
This is dull but I just wanted to get the PR up to get something started.   I have a much better idea to summarize the job and set the title that way, or let the user provide it in `submit` or something like that.  It should make tracking much easier.